packages/tokamak/contracts-bedrock/scripts/Deploy.s.sol

```diff
@@ -128,6 +135,13 @@ contract Deploy is Deployer {
         }
     }
 
+    /// @notice Deploy the Safe
+    function deployTON() public broadcast returns (address addr_) {
+        TON ton = new TON{ salt: keccak256(bytes(string("TON address"))) }();
+        addr_ = address(ton);
+
+    }
```

Comment:
I think it is better to add save deployment in case of deploying TON!
```suggestion
        save("TON", addr_);
```

Reply:
Thank you!
It makes sense